scripts/setup_iso.py

```diff
@@ -153,6 +153,9 @@ def determine_os_ver(srcdir, iso_info):
             print 'attempting LiveCD netboot'
             osname = 'LiveCD'
             osver = vid
+            for i in src_dir_list:
+                if 'PHOTON' in i:
+                    osname = 'PHOTON'
```

Comment:
are you suggesting code as below?

[...]

      elif 'LiveCD' in src_dir_list and 'isolinux' in src_dir_list: 
            print 'attempting LiveCD netboot'
            osname = 'LiveCD'
            osver = vid

      elif 'suse' in src_dir_list:
            osname = 'SUSE'
            if vid == '':
                assert ValueError, "The volume ID is not get correctly"
            ver_pattern = '(.+)-DVD-(.+)'
            m = re.search(ver_pattern, vid)
            assert m.group(1), "The version name is not retrieved correctly"
            osver = m.group(1)
        elif 'setup.exe' in src_dir_list:
            osname, osver = get_setup_exe_name_version(srcdir)
        elif 'PHOTON_VMWARE_CD' in src_dir_list:
            osname = "PHOTON"
            if vid == '':
               assert ValueError, "The volume ID is not get correctly"
           osver = vid.lstrip('PHOTON_')
[...]